images/base/files/etc/containerd/config.toml

```diff
@@ -17,6 +17,12 @@ version = 2
 [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
   # set default runtime handler to v2, which has a per-pod shim
   runtime_type = "io.containerd.runc.v2"
+  # base_runtime_spec is a file path to a JSON file with the OCI spec that will be used as the base spec that all
+  # container's are created from.
+  # Use containerd's `ctr oci spec > /etc/containerd/cri-base.json` to output initial spec file.
+  # Spec files are loaded at launch, so containerd daemon must be restarted on any changes to refresh default specs.
```

Comment:
We only need to justify why were are setting this, we can leave the rest to the upstream docs without repeating all of it here. We can also see what we're doing in the referenced file in the other sources.

Reply:
Done